pkgs/development/python-modules/para/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "para";
+  version = "0.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-RsMjKunY6p2IbP0IzdESiSICvthkX0C2JVWXukz+8hc=";
+  };
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  checkInputs = [ nose ];
```

Comment:
that should also be nativeCheckInputs

Reply:
good point! this actually raises a question i hadn't considered previously: is there ever a situation in which something should be in `checkInputs`? it seems like things should always just be in `nativeCheckInputs`?

Reply:
if you require extra libraries for the target architecture but I think its usecase is rather rare

Reply:
For me adding `nose` to `checkInputs` was a way to make the python package accessible (`import nose`) without triggering the `nose` _check hook_.

Reply:
But when cross compiling it should get the wrong arch for nose. Also pytestCheclHook should probably be higher priority. Just disable the nose hook if it is still happening.

Reply:
Addressed in #244666.